src/unstract/adapters/x2text/unstructured_community/src/static/json_schema.json

```diff
@@ -15,8 +15,8 @@
     "url": {
       "type": "string",
       "title": "URL",
-      "format": "url",
-      "default": "https://api.unstructured.io/general/v0/general",
+      "format": "uri",
+      "default": "http://unstract-unstructer-io:8000/general/v0/general",
```

Comment:
NIT: Have we intentionally named this service `unstructer` instead of `unstructured`?